dune-project

```diff
@@ -16,5 +16,6 @@
  (synopsis "Libraries and tools to keep watch on your OCaml lib's API changes")
  (depends
   (ocaml (and (>= 4.14.0) (< 5.0.0)))
+  (ppx_expect (and (>= v0.14.2) :with-test))
```

Comment:
Is there any specific reason why you added the `v0.14.2` lower bound?

Reply:
No, there isn't a reason for that. Would you prefer it without bounds?

Reply:
I took a quick look at the changelog of ppx_expect and it's not entirely clear to me. Let's write it without bound for now and see what the CI's lower bounds build says.

Reply:
It seems the lower bound build is broken anyway. Let's stick to the constraint less dep on ppx_expect for now. We'll update it later if needed!